frameworks/zsh4humans.zsh

```diff
@@ -0,0 +1,19 @@
+(
+
+emulate -L zsh -o err_return -o no_unset -o no_aliases
+cd -- $1
+# download zsh4humans v5
+curl -fsSLO https://github.com/romkatv/zsh4humans/archive/refs/heads/v5.tar.gz
+# copy over .zshenv and .zshrc
+tar -xzf v5.tar.gz
+cp zsh4humans-5/{.zshenv,.zshrc} ./
+rm -r v5.tar.gz zsh4humans-5
+# create a config for powerlevel10k
+>.p10k.zsh <<<$'POWERLEVEL9K_MODE=ascii\nPOWERLEVEL9K_DISABLE_CONFIGURATION_WIZARD=true'
+# install zimfw/git
+sed -i.bak -E 's|z4h install ohmyzsh/ohmyzsh|\0 zimfw/git|' .zshrc
```

Comment:
It's now failing with `zsh4humans/.zshrc:46: command not found: 0`. 

```suggestion
sed -i.bak -E 's|\(z4h install\) ohmyzsh/ohmyzsh|\0 zimfw/git|' .zshrc
```

Reply:
Also, looks like it's not compatible with [prezto-format modules](https://github.com/zimfw/zimfw/releases/tag/v1.4.3), right? We can manually add the proper code to the .zshrc file in this case. We had to add a similar code for antibody and zgen too.

Reply:
Originally I did the manual thing. I've now changed it to load zimfw via `z4h load`.